src/nomad_simulations/schema_packages/atoms_state.py

```diff
@@ -633,3 +666,35 @@ def normalize(self, archive: 'EntryArchive', logger: 'BoundLogger') -> None:
             self.chemical_symbol = self.resolve_chemical_symbol(logger=logger)
         if self.atomic_number is None:
             self.atomic_number = self.resolve_atomic_number(logger=logger)
+
+
+class AtomsState(Entity):
+    """
+    A base section to define each atom state information.
+    """
+
+    atom_definition_ref = Quantity(
+        type=AtomDefn,
+        description="""
+        A reference to the `ParticleDefn` section that describes this atom's species. This
+        reference avoids storing repeated elemental data (e.g., `chemical_symbol` and
+        `atomic_number`) in every atom, ensuring that all atoms of the same type link back
+        to a single shared definition.
+        """,
+    )
+
+    spin_definition_ref = Quantity(
+        type=SpinDefn, description='Reference to the spin definition for this atom.'
+    )
+
+    orbitals_state = SubSection(sub_section=OrbitalsState.m_def, repeats=True)
+
+    core_hole = SubSection(sub_section=CoreHole.m_def, repeats=False)
+
+    hubbard_interactions = SubSection(
+        sub_section=HubbardInteractions.m_def, repeats=False
+    )
+
+
+class ParticleState(Entity):
```

Comment:
I will create a new MR to integrate/migrate @Bernadette-Mohr 's previous work on the particle/CG representation after this one is done.